src/briefcase/platforms/iOS/xcode.py

```diff
@@ -12,7 +12,7 @@
 )
 from briefcase.config import BaseConfig
 from briefcase.console import select_option
-from briefcase.exceptions import BriefcaseCommandError
+from briefcase.exceptions import BriefcaseCommandError, InvalidDeviceError
```

Comment:
Unrelated to this PR :P but OK :)

Reply:
Tangentially related... I introduced an InvalidDeviceError, so I'm making sure it's used consistently.